From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Mark Salyzyn <salyzyn@google.com>
Date: Tue, 3 Sep 2019 13:17:23 -0700
Subject: ANDROID: GKI: enable CONFIG_TIPC for x86

Adds compile coverage for TIPC in the TH builds, runtime not so much.

Signed-off-by: Mark Salyzyn <salyzyn@google.com>
Bug: 140406060
Change-Id: I7731157396372683c906f1f4eb2fdbdb7015f446
---
 arch/x86/configs/gki_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/x86/configs/gki_defconfig b/arch/x86/configs/gki_defconfig
index 9b76d55e8c41..6b0ff4ca236a 100644
--- a/arch/x86/configs/gki_defconfig
+++ b/arch/x86/configs/gki_defconfig
@@ -145,6 +145,7 @@ CONFIG_IP6_NF_FILTER=y
 CONFIG_IP6_NF_TARGET_REJECT=y
 CONFIG_IP6_NF_MANGLE=y
 CONFIG_IP6_NF_RAW=y
+CONFIG_TIPC=y
 CONFIG_L2TP=y
 CONFIG_NET_SCHED=y
 CONFIG_NET_SCH_HTB=y
